files/json2timeseriesdb

```diff
@@ -240,6 +240,35 @@ def influx_tag_parser(tag)
     tag.delete('broker-service')
   end
 
+  if tag.include?('postgres') && tag.include?('databases')
+    n = tag.index('databases')
+    db_name = tag[n + 1]
+    tag_set = "#{tag_set}database=#{db_name},"
+    tag.delete_at(n + 1)
+  end
+
+  if tag.include?('postgres') && tag.include?('replication_slots')
+    n = tag.index('replication_slots')
+    slot_name = tag[n + 1]
+    tag_set = "#{tag_set}replication_slot=#{slot_name}"
+    tag.delete_at(n + 1)
+  end
+
+  if tag.include?('postgres') && tag.include?('replication_subs')
+    n = tag.index('replication_subs')
+    slot_name = tag[n + 1]
+    tag_set = "#{tag_set}replication_slot=#{slot_name}"
```

Comment:
Should this be `"#{tag_set}replication_subs=#{slot_name}"`?

Reply:
The same "slot name" is used on both the primary side with slots and the replica side with subscriptions, so I decided to use the same tag name for both.